contrib/go-pg/go-pg/pg_go.go

```diff
@@ -0,0 +1,54 @@
+package pggo
+
+import (
+	"context"
+	"github.com/go-pg/pg/v10"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+	"time"
+)
+
+const (
+	gopgStartSpan  = "dd-trace-go:span"
+	//gopgConfig  = "dd-trace-go:config"
+)
+
+func Hook(db *pg.DB) *pg.DB {
+
+	db.AddQueryHook(&QueryHook{})
+	return db
+}
+
+type QueryHook struct {}
+
+func (h *QueryHook) BeforeQuery(ctx context.Context,qe *pg.QueryEvent) (context.Context, error){
+	if qe.Stash == nil{
+		qe.Stash = make(map[interface{}]interface{})
+	}
+	qe.Stash[gopgStartSpan] = time.Now()
```

Comment:
Do we need to use this? Just start the span here and use the `ctx` to propagate it and end it in `AfterQuery`.